### DISCUSSION TITLE
Allow characters other than default mc ones in player names to be usable?

### DISCUSSION THREAD

**Discussion by vijaypondini** [GriefPrevention/GriefPrevention#1538] [Ideas]
Ok weird title yes, but i'm currently running a geysermc server, and i first, set the prefix to `.` , however after that the plugin mostly won't work with stuff like permission trust, /trust etc. 
Example name `.QuickestRhyme56`
`/trust .QuickestRhyme56` says player added, but when we do `/trustlist` it shows
`.quickestrhyme56` and i also don't actually get trusted. 
However switching to _ which is a usable mc character in names, it would work.

So is it possible to allow unsafe usernames like this?

**Comment by Jikoo** [comment]
See #1360 - abnormal characters are fine, but periods are not because those are interpreted as permissions.

**Comment by vijaypondini** [comment]
So is this planned to be updated anytime soon to not interpret strings as permissions?

**Reply by Jikoo** [reply]
You're free to submit a PR, but I doubt flat out feature removal would be accepted. You'd need to offer the option to check first for a player, then fall through to adding as a permission.

It's probably a lot easier for you to use a better prefix character, like the default `*` or a nice `+`.

**Reply by tmantti** [reply]
Period is the new default prefix on Floodgate 2.0 so this breaks support with most geyser installations unless config changes are made.

**Reply by vijaypondini** [reply]
Yeah, tbh that. Many many ppl will be using `.` and * and + is a pain since you need to wrap double quotes around it

**Reply by Jikoo** [reply]
Unfortunate. Understandable that they moved away from `*` for plugins that support glob matching, but a period isn't really a better solution on their end because it's largely reserved for permissions :\

I don't understand what the problem with a prefix like `+` is, what do you mean about double quotes? If you're just talking about putting special characters in YAML strings, quoting is a basic thing that any server owner should already know or they're gonna have a very rude awakening when Bukkit's YAML system wipes out their invalid config syntax.

If we could limit ideas and discussion to one place, that would be a lot easier. I personally am going to unsubscribe from this thread - if you want to discuss it further, I am subscribed to the linked issue.
